prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Andrew Hoang';
+    console.log(fullName);
+
+    var numberOfPets = '0';
+    console.log(numberOfPets);
```

Comment:
We want to leave the quote marks off of our number values so that the JavaScript engine doesn't interpret them as strings.

```js
var numberOfPets = 0;
```